kbase-extension/static/kbase/js/common/cellComponents/tabs/jobStatus/jobStatusTable.js

```diff
@@ -150,10 +154,15 @@ define([
                         {
                             className: `${cssBaseClass}__cell--object`,
                             render: (data, type, row) => {
-                                const params = jobManager.model.getItem(
-                                    `exec.jobs.params.${row.job_id}`
-                                );
-                                return params ? renderParams(params) : row.job_id;
+                                const params = jobManager.model.getItem('exec.jobs.params') || {};
+
+                                if (row.retry_parent && params[row.retry_parent]) {
```

Comment:
`row` here is the jobState object for this table row. If the job has a `retry_parent`, use the data for that to render the object description; if there are params for the job itself, use them; otherwise, just show the job ID.